message/validation/message_counts.go

```diff
@@ -39,34 +37,28 @@ func (c *MessageCounts) ValidateConsensusMessage(signedSSVMessage *spectypes.Sig
 	switch msg.MsgType {
 	case specqbft.ProposalMsgType:
 		if c.Proposal >= limits.Proposal {
-			err := ErrTooManySameTypeMessagesPerRound
+			err := ErrDuplicatedMessage
 			err.got = fmt.Sprintf("proposal, having %v", c.String())
 			return err
 		}
 	case specqbft.PrepareMsgType:
 		if c.Prepare >= limits.Prepare {
-			err := ErrTooManySameTypeMessagesPerRound
+			err := ErrDuplicatedMessage
 			err.got = fmt.Sprintf("prepare, having %v", c.String())
 			return err
 		}
 	case specqbft.CommitMsgType:
 		if len(signedSSVMessage.GetOperatorIDs()) == 1 {
 			if c.Commit >= limits.Commit {
-				err := ErrTooManySameTypeMessagesPerRound
+				err := ErrDuplicatedMessage
```

Comment:
duplicate is more correct because there should be only one message per signer per consensus per round?

Reply:
We don't check decided message count here anymore because other checks override it, so if we'd keep the check here, we wouldn't be able to cover it with tests. Therefore this method checks only duplicates now. Also, it's `ErrDuplicatedMessage` in the knowledge base and we want to align code and KB as much as possible